src/Question.js

```diff
@@ -9,10 +9,11 @@ class Question extends Component {
     answers: PropTypes.arrayOf(PropTypes.shape({
       option: PropTypes.string.isRequired,
       correct: PropTypes.bool.isRequired
-    }))
+    })),
+    count: PropTypes.number.isRequired
   }
   handleClick(answer) {
-    const itsRight = answer.correct; 
+    const itsRight = answer.correct;
     if (itsRight) {
       alert('this is correct!')
```

Comment:
Notice that we pass in a boolean for whether the answer is correct or not in handleClick. Since handleAnswerSelected will replace handleClick it will receive the same argument!